news/2023/2023-07-05-beatmap-pack-new-categories.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
+date: 2023-07-05 09:00:00 +0000
+---
+
+Check out the latest and greatest new beatmap pack categories!
+
+![](https://chromb.uk/personal/packsbanner.png)
```

Comment:
put this in the repo as `wiki/shared/news/2023-07-{day}-introducing-new-beatmap-pack-categories/banner.png`